include/poncos/vm_controller.hpp

```diff
@@ -23,6 +23,10 @@
 #include "poncos/poncos.hpp"
 
 #include <fast-lib/mqtt_communicator.hpp>
+#include <fast-lib/message/migfra/pci_id.hpp>
```

Comment:
No, we need both, the `task` and the `result`. However, the following is obsolete and I'll remove this: